pkg/kubelet/util/manager/watch_based_manager.go

```diff
@@ -42,8 +42,8 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 )
 
-type listObjectFunc func(string, string, metav1.ListOptions) (runtime.Object, error)
-type watchObjectFunc func(string, string, metav1.ListOptions) (watch.Interface, error)
+type listObjectFunc func(string, string, int, metav1.ListOptions) (runtime.Object, error)
```

Comment:
Can we add comment or name for arguments?

Reply:
will add arg names.